{"name": "test_sign_up_negative_8[3]", "status": "failed", "statusDetails": {"message": "AssertionError: assert \"{'username':...must match']}\" == 'Incorrect us...ds must match'\n  - Incorrect username length and passwords must match\n  + {'username': ['Incorrect username length'], 'password': ['Passwords must match']}", "trace": "self = <test_ui.TestUI object at 0x7fd5b03b4cd0>\nget_fake_user_info = ('wpetrova', '%9M+Kr71+7', 'alevtina_1997@rusenergoresurs.com')\nlen_username = 3\nhome_page_object = <ui.pages.home_page.HomePage object at 0x7fd5b03b52b0>\nreg_page_object = <ui.pages.reg_page.RegPage object at 0x7fd5b03b5df0>\n\n    @allure.feature('UI tests')\n    @allure.story('Sign up page story')\n    @pytest.mark.parametrize('len_username', [1, 2, 3, 4, 5])\n    def test_sign_up_negative_8(self, get_fake_user_info, len_username, home_page_object, reg_page_object):\n        \"\"\"Проверка регистрации пользователя с одновременно несовпадающими полями password, confirm_password и невалидным полем username\"\"\"\n        self.logger.start_logging(\n            f'./project/app_logs/{inspect.getframeinfo(inspect.currentframe()).function}_ui_log_{len_username}.txt')\n        with allure.step('Получаем информацию о невалидном пользователе'):\n            username, password, email = get_fake_user_info\n            if len(username) < len_username:\n                username *= len_username\n            username = username[:len_username]\n        with allure.step('Получаем объект страницы регистрации'):\n            reg_page = self.login_page.sign_up(reg_page_object)\n        with allure.step('Регистрируемся и проверяем валидное отбражение ошибки на странице регистрации'):\n            try:\n>               reg_page.sign_up(username, email, password, home_page_object, wrong_password=password + 'qwe')\n\nemail      = 'alevtina_1997@rusenergoresurs.com'\nget_fake_user_info = ('wpetrova', '%9M+Kr71+7', 'alevtina_1997@rusenergoresurs.com')\nhome_page_object = <ui.pages.home_page.HomePage object at 0x7fd5b03b52b0>\nlen_username = 3\npassword   = '%9M+Kr71+7'\nreg_page   = <ui.pages.reg_page.RegPage object at 0x7fd5b03b5df0>\nreg_page_object = <ui.pages.reg_page.RegPage object at 0x7fd5b03b5df0>\nself       = <test_ui.TestUI object at 0x7fd5b03b4cd0>\nusername   = 'wpe'\n\nproject/tests/test_ui.py:347: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <ui.pages.reg_page.RegPage object at 0x7fd5b03b5df0>, username = 'wpe'\nemail = 'alevtina_1997@rusenergoresurs.com', password = '%9M+Kr71+7'\nhome_page_object = <ui.pages.home_page.HomePage object at 0x7fd5b03b52b0>\nwrong_password = '%9M+Kr71+7qwe'\n\n    def sign_up(self, username, email, password, home_page_object, wrong_password=None):\n        self.paste_sign_up_info(username, email, password, wrong_password=wrong_password)\n        self.click_accept_reg_button()\n        self.click_submit_button()\n        try:\n            if wait(func=self.check_available_of_element(self.locators.POP_UP_ERR_FIELD, auto_use=False), timeout=1):\n>               raise InvalidRegInfoException(self.find(self.locators.POP_UP_ERR_FIELD).text)\nE               ui.utils.exceptions.InvalidRegInfoException: {'username': ['Incorrect username length'], 'password': ['Passwords must match']}\n\nemail      = 'alevtina_1997@rusenergoresurs.com'\nhome_page_object = <ui.pages.home_page.HomePage object at 0x7fd5b03b52b0>\npassword   = '%9M+Kr71+7'\nself       = <ui.pages.reg_page.RegPage object at 0x7fd5b03b5df0>\nusername   = 'wpe'\nwrong_password = '%9M+Kr71+7qwe'\n\nproject/ui/pages/reg_page.py:31: InvalidRegInfoException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <test_ui.TestUI object at 0x7fd5b03b4cd0>\nget_fake_user_info = ('wpetrova', '%9M+Kr71+7', 'alevtina_1997@rusenergoresurs.com')\nlen_username = 3\nhome_page_object = <ui.pages.home_page.HomePage object at 0x7fd5b03b52b0>\nreg_page_object = <ui.pages.reg_page.RegPage object at 0x7fd5b03b5df0>\n\n    @allure.feature('UI tests')\n    @allure.story('Sign up page story')\n    @pytest.mark.parametrize('len_username', [1, 2, 3, 4, 5])\n    def test_sign_up_negative_8(self, get_fake_user_info, len_username, home_page_object, reg_page_object):\n        \"\"\"Проверка регистрации пользователя с одновременно несовпадающими полями password, confirm_password и невалидным полем username\"\"\"\n        self.logger.start_logging(\n            f'./project/app_logs/{inspect.getframeinfo(inspect.currentframe()).function}_ui_log_{len_username}.txt')\n        with allure.step('Получаем информацию о невалидном пользователе'):\n            username, password, email = get_fake_user_info\n            if len(username) < len_username:\n                username *= len_username\n            username = username[:len_username]\n        with allure.step('Получаем объект страницы регистрации'):\n            reg_page = self.login_page.sign_up(reg_page_object)\n        with allure.step('Регистрируемся и проверяем валидное отбражение ошибки на странице регистрации'):\n            try:\n                reg_page.sign_up(username, email, password, home_page_object, wrong_password=password + 'qwe')\n            except InvalidRegInfoException as exception:\n                allure.attach(self.driver.get_screenshot_as_png(), 'Error screen', allure.attachment_type.PNG)\n                self.logger.stop_logging()\n                allure.attach(self.logger.get_log_info(), 'App log', allure.attachment_type.TEXT)\n>               assert exception.args[0] == 'Incorrect username length and passwords must match'\nE               assert \"{'username':...must match']}\" == 'Incorrect us...ds must match'\nE                 - Incorrect username length and passwords must match\nE                 + {'username': ['Incorrect username length'], 'password': ['Passwords must match']}\n\nemail      = 'alevtina_1997@rusenergoresurs.com'\nget_fake_user_info = ('wpetrova', '%9M+Kr71+7', 'alevtina_1997@rusenergoresurs.com')\nhome_page_object = <ui.pages.home_page.HomePage object at 0x7fd5b03b52b0>\nlen_username = 3\npassword   = '%9M+Kr71+7'\nreg_page   = <ui.pages.reg_page.RegPage object at 0x7fd5b03b5df0>\nreg_page_object = <ui.pages.reg_page.RegPage object at 0x7fd5b03b5df0>\nself       = <test_ui.TestUI object at 0x7fd5b03b4cd0>\nusername   = 'wpe'\n\nproject/tests/test_ui.py:352: AssertionError"}, "description": "Проверка регистрации пользователя с одновременно несовпадающими полями password, confirm_password и невалидным полем username", "steps": [{"name": "Получаем информацию о невалидном пользователе", "status": "passed", "start": 1590595576022, "stop": 1590595576022}, {"name": "Получаем объект страницы регистрации", "status": "passed", "start": 1590595576022, "stop": 1590595576179}, {"name": "Регистрируемся и проверяем валидное отбражение ошибки на странице регистрации", "status": "failed", "statusDetails": {"message": "AssertionError: assert \"{'username':...must match']}\" == 'Incorrect us...ds must match'\n  - Incorrect username length and passwords must match\n  + {'username': ['Incorrect username length'], 'password': ['Passwords must match']}\n", "trace": "  File \"/media/qunity/Workspace/Python_projects/qa-project/project/tests/test_ui.py\", line 352, in test_sign_up_negative_8\n    assert exception.args[0] == 'Incorrect username length and passwords must match'\n"}, "attachments": [{"name": "Error screen", "source": "e8597c69-0ce7-4833-8117-378ad702589d-attachment.png", "type": "image/png"}, {"name": "App log", "source": "f2808577-7aef-4d06-b999-d09dd8c8474a-attachment.txt", "type": "text/plain"}], "start": 1590595576179, "stop": 1590595578085}], "parameters": [{"name": "len_username", "value": "3"}], "start": 1590595575930, "stop": 1590595578085, "uuid": "19cd14a8-5dbd-41ba-8139-05db7402064f", "historyId": "41e0e7adc693917e7d151925c584ef53", "testCaseId": "b16ff2d1f506ca04887854c5a80bcb6f", "fullName": "tests.test_ui.TestUI#test_sign_up_negative_8", "labels": [{"name": "feature", "value": "UI tests"}, {"name": "story", "value": "Sign up page story"}, {"name": "tag", "value": "UI"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_ui"}, {"name": "subSuite", "value": "TestUI"}, {"name": "host", "value": "dmaksimov-laptop"}, {"name": "thread", "value": "22050-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_ui"}]}